src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -127,13 +130,49 @@ def train(
             )
             for entry in training_data
         ]
-        df = pd.concat(dfs)
+        if validation_data is not None or last_k_for_val is not None:
+            self.kwargs["auto_stack"] = False
+            logging.warning(
```

Comment:
It is recommended to create a specific logger object for this module, instead of using the default one from the `logging` module. Just add this on top after the imports

```
logger = logging.getLogger(__name__)
```

and use `logger.log(…)` throughout the code in this module